CHANGELOG.md

```diff
@@ -73,6 +73,8 @@
 ## Blocksconvert
 
 * [ENHANCEMENT] Builder: add `-builder.timestamp-tolerance` option which may reduce block size by rounding timestamps to make difference whole seconds. #3891
+* [ENHANCEMENT] Scanner: add support for DynamoDB (v9 schema only). #3828
```

Comment:
Could you rebase `master` and move this under the "master / unreleased", please?

Reply:
done